src/Directory.Build.targets

```diff
@@ -0,0 +1,4 @@
+<Project>
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
+  <Import Project="$(MSBuildThisFileDirectory)/../eng/ProjectStaging.targets" />
```

Comment:
Would it better to add the target here directly?

Reply:
I'm not quite sure what you mean... Could you please clarify?